packages/desktop-gui/cypress/integration/project_nav_spec.js

```diff
@@ -372,7 +372,7 @@ describe('Project Nav', function () {
 
     describe('browser with info', function () {
       beforeEach(function () {
-        this.info = 'The Electron browser is the version of Chrome that is bundled with Electron. Cypress uses this browser when running headlessly, so it may be useful for debugging issues that occur only in headless mode.'
+        this.info = 'foo info bar'
```

Comment:
nice

Reply:
it was getting in the way of my search for headless, realized this doesn't need to be a real string